include/up-cpp/uri/builder/BuildUAuthority.h

```diff
@@ -45,7 +46,7 @@ namespace uprotocol::uri {
         uprotocol::v1::UAuthority authority_;
```

Comment:
Consider documenting the overall purpose of the BuildUAuthority class and its intended usage

Reply:
Will address in a separate PR as that is a pre-existing component. Issue #85 has been filed to track this request.